etl/steps/data/garden/explorers/2021/food_explorer.regions.json

```diff
@@ -0,0 +1 @@
China", "Christmas Island", "Cocos Islands", "Cyprus", "Georgia", "Hong Kong", "India", "Indonesia", "Iran", "Iraq", "Israel", "Japan", "Jordan", "Kazakhstan", "Korea", "Kuwait", "Kyrgyzstan", "Laos", "Lebanon", "Macao", "Malaysia", "Maldives", "Mongolia", "Myanmar", "Nepal", "North Korea", "Oman", "Pakistan", "Palestine", "Philippines", "Qatar", "Republic of Vietnam", "Saudi Arabia", "Singapore", "South Korea", "Sri Lanka", "Syria", "Taiwan", "Tajikistan", "Thailand", "Timor", "Turkmenistan", "United Arab Emirates", "Uzbekistan", "Vietnam", "Yemen", "Yemen Arab Republic", "Yemen People's Republic"], "Europe": ["Aland Islands", "Albania", "Andorra", "Austria", "Austria-Hungary", "Baden", "Bavaria", "Belarus", "Belgium", "Bosnia and Herzegovina", "Bulgaria", "Channel Islands", "Croatia", "Czechia", "Czechoslovakia", "Denmark", "East Germany", "Estonia", "Faeroe Islands", "Finland", "France", "Germany", "Gibraltar", "Greece", "Guernsey", "Hanover", "Hesse Electoral", "Hesse Grand Ducal", "Hungary", "Iceland", "Ireland", "Isle of Man", "Italy", "Jersey", "Kosovo", "Latvia", "Liechtenstein", "Lithuania", "Luxembourg", "Malta", "Mecklenburg Schwerin", "Modena", "Moldova", "Monaco", "Montenegro", "Netherlands", "North Macedonia", "Northern Cyprus", "Norway", "Parma", "Poland", "Portugal", "Romania", "Russia", "San Marino", "Saxony", "Serbia", "Serbia and Montenegro", "Slovakia", "Slovenia", "Spain", "Svalbard and Jan Mayen", "Sweden", "Switzerland", "Turkey", "Tuscany", "Two Sicilies", "Ukraine", "United Kingdom", "USSR", "Vatican", "West Germany", "Wuerttemburg", "Yugoslavia"], "South America": ["Argentina", "Bolivia", "Brazil", "Caribbean Netherlands", "Chile", "Colombia", "Ecuador", "Falkland Islands", "French Guiana", "Guyana", "Paraguay", "Peru", "Suriname", "Uruguay", "Venezuela"], "North America": ["Anguilla", "Antigua and Barbuda", "Aruba", "Bahamas", "Barbados", "Belize", "Bermuda", "Bonaire Sint Eustatius and Saba", "British Virgin Islands", "Canada", "Cayman Islands", "Costa Rica", "Cuba", "Curacao", "Dominica", "Dominican Republic", "El Salvador", "Greenland", "Grenada", "Guadeloupe", "Guatemala", "Haiti", "Honduras", "Jamaica", "Martinique", "Mexico", "Montserrat", "Netherlands Antilles", "Nicaragua", "Panama", "Puerto Rico", "Saint Barthlemy", "Saint Kitts and Nevis", "Saint Lucia", "Saint Martin (French part)", "Saint Pierre and Miquelon", "Saint Vincent and the Grenadines", "Sint Maarten (Dutch part)", "Trinidad and Tobago", "Turks and Caicos Islands", "United States", "United States Virgin Islands"], "Oceania": ["American Samoa", "Australia", "Cook Islands", "Fiji", "French Polynesia", "Guam", "Kiribati", "Marshall Islands", "Melanesia", "Micronesia (country)", "Nauru", "New Caledonia", "New Zealand", "Niue", "Norfolk Island", "Northern Mariana Islands", "Palau", "Papua New Guinea", "Pitcairn", "Polynesia", "Samoa", "Solomon Islands", "Tokelau", "Tonga", "Tuvalu", "United States Minor Outlying Islands", "Vanuatu", "Wallis and Futuna"]}
```

Comment:
This one will be easier to maintain and QA if you reformat it (e.g. with `indent=2` in Python).

Reply:
Thanks!